Virtual Keyboard/Assets/Scripts/CategoryManager.cs

```diff
@@ -247,4 +247,24 @@ public class CategoryManager : MonoBehaviour {
 		}
 	}
 
+	[ContextMenu("Add New Category")]
+	void addCategory() {
+		Debug.Log ("Add New Category");
+
+		GameObject newCategory = new GameObject ();
+		newCategory.transform.SetParent (transform);
+		newCategory.name = "New Category";
```

Comment:
You can just set the title on the ListManger, it auto updates gameObject.name making this redundent